src/CoreShop/Bundle/ProductBundle/CoreExtension/ProductSpecificPriceRules.php

```diff
@@ -514,4 +518,24 @@ private function getConfigConditions()
     {
         return $this->getContainer()->getParameter('coreshop.product_specific_price_rule.conditions');
     }
+
+    public function getParameterTypeDeclaration(): ?string
```

Comment:
can you remove this again? we don't support that in 2.2

Reply:
Done, thanks for your support!